### PR TITLE
Coprarna2

### PR DESCRIPTION
**Pull request by eggzilla**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by eggzilla** [comment]
Please do NOT merge, some changes are still missing!

File: recipes/coprarna/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  build:
+    - perl >=5.22
+    - perl-bioperl >=1.6.924
+    - perl-list-moreutils ==0.413
+    - perl-parallel-forkmanager ==1.17
+    - perl-getopt-long
+    - perl-bio-eutilities ==1.75
+    - r-base >=3.2.2
+    - r-seqinr
+    - python ==2.7.13
```
Comment (bgruening) [inline recipes/coprarna/meta.yaml]: python should not be pinned like this, please skip python 3 in the build section

File: recipes/coprarna/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+# commented versions are tested as working
+requirements:
+  build:
+    - perl >=5.22
+    - perl-bioperl >=1.6.924
+    - perl-list-moreutils ==0.413
+    - perl-parallel-forkmanager ==1.17
+    - perl-getopt-long
+    - perl-bio-eutilities ==1.75
+    - r-base >=3.2.2
```
Comment (bgruening) [inline recipes/coprarna/meta.yaml]: R pinning is also not needed.

File: recipes/coprarna/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+{% set version = "alpha2" %}
+{% set md5 = "95abc9567e995331c6412f9b2f3ac6aa" %} 
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.zip
+  url: https://github.com/eggzilla/CopraRNA/archive/{{ version }}.zip
```
Comment (bgruening) [inline recipes/coprarna/meta.yaml]: should this be Patricks repo or Backofenlab?

File: recipes/coprarna/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    - sed
+    - gawk
+    - bzip2
+
+test:
+  commands:
+    - CopraRNA2.pl --help
+    
+about:
+  home: http://rna.informatik.uni-freiburg.de/CopraRNA/Input.jsp
```
Comment (bgruening) [inline recipes/coprarna/meta.yaml]: I guess the github repo is more appropriate ...

File: recipes/coprarna/meta.yaml (hunk 3 above)
Comment (eggzilla) [reply recipes/coprarna/meta.yaml]: yes, this is only for testing until patrick has tagged the final 2.x release

**Comment by PatrickRWright** [comment]
I think there is no need to specifically tag the IntaRNA version. i.e. 2.1.0 is working fine.

Also the example can be made faster by setting -ntup 75 -ntdown 75

**Comment by PatrickRWright** [comment]
Thanks @eggzilla

**Pull request merged**